devops/ansible-role-girder/molecule/ansible-module/playbook.yml

```diff
@@ -3,24 +3,9 @@
   hosts: all
 
   vars:
-    girder_virtualenv: "{{ ansible_user_dir }}/.venvs/girder"
+    girder_virtualenv: "{{ ansible_user_dir }}/.virtualenvs/girder"
```

Comment:
Is there any reason to define these in the tests anymore?

Reply:
In this playbook, it's because we reference `girder_virtualenv` later in `post_tasks`. In other playbooks, where we don't need to work with `girder_virtualenv` after the role completes, we accept the role default.